src/infrastructure/middleware/authorization/jwt/index.ts

```diff
@@ -14,27 +14,29 @@ const TOKEN_PARTS = 2;
 const TOKEN_POSITION_VALUE = 1;

```

Comment:
Please move it inside the class context and use static as well.

Reply:
That's not my code, it's more than 2 years old hauauhah

I will adjust here